02_classify_mutations/nbconverted/new_vs_old_genes.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import mpmp.config as cfg
+import mpmp.utilities.analysis_utilities as au
+import mpmp.utilities.data_utilities as du
+import mpmp.utilities.plot_utilities as plu
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+
+# In[2]:
+
+
+# location of classification results
+results_dir = Path(cfg.results_dirs['mutation'], 'merged_all', 'gene').resolve()
+
+# set significance cutoff after FDR correction
+SIG_ALPHA = 0.001
+
+
+# ### Load results and perform statistical testing/correction
+
+# In[3]:
+
+
+# load raw data
+results_df = au.load_stratified_prediction_results(results_dir, 'gene')
+
+# here we want to use compressed data for methylation datasets (27k and 450k)
+# the results in 02_classify_compressed/compressed_vs_raw_results.ipynb show that
```

Comment:
This is a helpful note, but also I'm not sure where the 02_classify_compressed directory is. Did it get renamed?

Reply:
Yeah, this is out of date - I'll update it.